modules/components/forms/text_input.cjsx

```diff
@@ -21,6 +21,7 @@ module.exports = class TextInput extends RestFormElement
 
   @defaultProps =
     inputType: 'text'
+    value:     ''
```

Comment:
Isn't it a little strange to have a value of '' if its a number input? or a select?